test.py

```diff
@@ -643,6 +643,14 @@ def get_test_suite(self) -> Optional[ET.Element]:
 
     def __parse_logger(self) -> None:
         def adjust_suite_name(name):
+            if name == 'Master Test Suite':
+                # the test does not define BOOST_TEST_MODULE, let's give it
+                # a better name by translating "test/boost/sstable_directory_test.cc"
+                # to "boost.sstable_directory_test"
```

Comment:
Excellent. I wish that all boost tests would just be called "boost.*" - why do we need to honor the "name" at all, even if it's not "Master Test Suite"?

Reply:
fixed in the latest revision as well.